### ISSUE TITLE
Send to Draft not working

### ISSUE THREAD

**Issue by devinbalkind** [PressForward/pressforward#642]
Send to Draft isn't working.

I'm trying to "Send to Draft" via the "All Content" section.  The button works.  When I click Send to Draft it will automatically nominate the article.  When I click again the "Send to Draft" text is highlighted - indicating it worked.  But no draft appears in the WordPress posts section.

When I click on the story from All Content and am viewing it in the popup window, it behaves the same way and also doesn't place the content into the WordPress post.

When I first set up the site the Send to Draft feature was working sporadically but now doesn't work at all.  I just implemented SSL which I don't think has any effect on this, but thought I'd mention it.

Please help.

Very excited to start using this amazing looking plugin.

**Comment by eolson3** [comment]
Same thing happens on our test site.  Any idea of what may be causing this, Aram?


Eric

**Comment by devinbalkind** [comment]
Please help.  Without the "Send to ___" function the plugin is useless and my project can't launch. 

Thanks.

**Comment by AramZS** [comment]
Will investigate and resolve this asap

**Comment by devinbalkind** [comment]
Thanks so so much!

**Comment by AramZS** [comment]
@devinbalkind @eolson3 What are the settings in your PressForward Preferences -> Site Options bottom area? Should look like:
![image](https://cloud.githubusercontent.com/assets/748069/9864369/ded703e8-5b16-11e5-995e-d2710379b7c3.png)

**Comment by devinbalkind** [comment]
Post Status: Draft

Post Type: Post

![screen shot 2015-09-14 at 5 45 46 pm](https://cloud.githubusercontent.com/assets/444311/9865259/983ae822-5b08-11e5-8efd-441c73c3802b.png)

**Comment by AramZS** [comment]
Hi @devinbalkind, I have duplicated these settings on my test and live environments and have been unable to duplicate this error. @eolson3 You said this is happening on your test site as well? Is this pfsample?

**Comment by AramZS** [comment]
@devinbalkind @eolson3 Can you tell me if you see an error in the console? https://developer.chrome.com/devtools/docs/console

**Comment by devinbalkind** [comment]
No errors but a bunch of warnings involving HTTPS.

Pasted below:

admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13413/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13413&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
8admin.php?page=pf-menu:1 Consider using 'dppx' units, as in CSS 'dpi' means dots-per-CSS-inch, not dots-per-physical-inch, so does not correspond to the actual 'dpi' of a screen. In media query expression: print, (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13404/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13404&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13393/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13393&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13399/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13399&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13397/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13397&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13395/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13395&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13386/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13386&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13386/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13386&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13379/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13379&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13374/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13374&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13366/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13366&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13328/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13328&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
views.js?ver=4.3.1:458 Waiting for load.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13347/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13347&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13360/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13360&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13351/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13351&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13356/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:139 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13356&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:511 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13413/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:511 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13413&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:711 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13404/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:711 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13404&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:813 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13393/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:813 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13393&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:925 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13399/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:925 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13399&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1023 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13397/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1023 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13397&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1126 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13395/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1126 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13395&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1224 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13386/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1224 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13386&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1324 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13386/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1324 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13386&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1423 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13379/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1423 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13379&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1534 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13374/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1534 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13374&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1741 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13366/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1741 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13366&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1842 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13328/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1842 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13328&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1941 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13347/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:1941 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13347&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2049 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13360/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2049 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13360&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2150 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13351/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2150 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13351&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2253 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://feeds.wordpress.com/1.0/comments/recoverydiva.wordpress.com/13356/'. This content should also be served over HTTPS.
admin.php?page=pf-menu:2253 Mixed Content: The page at 'https://resiliencecolab.org/wp-admin/admin.php?page=pf-menu' was loaded over HTTPS, but requested an insecure image 'http://pixel.wp.com/b.gif?host=recoverydiva.com&blog=12534994&post=13356&subd=recoverydiva&ref=&feed=1'. This content should also be served over HTTPS.
views.js?ver=4.3.1:463 Load complete.

I turned off my SSL plugin and started using the site at HTTP but that didn't resolve the problem.  

Looking at the  "Nominate" page I get the following errors:

8 Consider using 'dppx' units, as in CSS 'dpi' means dots-per-CSS-inch, not dots-per-physical-inch, so does not correspond to the actual 'dpi' of a screen. In media query expression: print, (-webkit-min-device-pixel-ratio: 1.25), (min-resolution: 120dpi)
views.js?ver=4.3.1:458 Waiting for load.
jquery.infinitescroll.js?ver=4.3.1:226 Uncaught TypeError: Cannot read property 'debug' of undefinedjquery.infinitescroll.js?ver=4.3.1:226 infscr_debugjquery.infinitescroll.js?ver=4.3.1:486 infscr_validatejquery.infinitescroll.js?ver=4.3.1:114 infscr_createjquery.infinitescroll.js?ver=4.3.1:23 infscrjquery.infinitescroll.js?ver=4.3.1:744 (anonymous function)load-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 m.extend.eachload-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 m.fn.m.eachjquery.infinitescroll.js?ver=4.3.1:732 infscr_initscroll-imp.js?ver=4.3.1:6 (anonymous function)load-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 jload-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 k.fireWithload-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 m.extend.readyload-scripts.php?c=1&load[]=jquery-core,jquery-migrate,utils,jquery-ui-core,jquery-ui-widget,jquery…:2 J
views.js?ver=4.3.1:463 Load complete.

Thanks for looking into this.

**Comment by AramZS** [comment]
@devinbalkind What happens in that console when you try to use the Send to Draft button?

**Comment by devinbalkind** [comment]
Nothing when I click to "Send to Draft".  If I click it a second time I get
"This is already active." message in the console but no error.

**Comment by lmrhody** [comment]
Check for duplicate post and follow Eric to see what he's seeing.

**Comment by eolson3** [comment]
Resolved on the test site.

**Comment by bschnei6** [comment]
Issue verified. I used the "Send to draft" from the all content page. A loading bar appeared and failed to resolve. I went to check the "Posts" tab and nothing had been sent to draft. I returned to the the "All Content" page and the items had the nominate button turned green. The items appeared in the nominations. This is the error that appeared when the loading bar failed to resolve: 

Uncaught Error: Syntax error, unrecognized expression: Catchable fatal error: Object of class stdClass could not be converted to string in /websites/pftesting/www/wp-content/plugins/pressforward/includes/functions.php on line 805

![screen shot 2015-10-07 at 10 10 31 am](https://cloud.githubusercontent.com/assets/5577064/10339914/b03aaf36-6cdb-11e5-83fa-7e9d8e0d35a5.png)

**Comment by bschnei6** [comment]
This issue appears to be related to the particular feed that the content is being drafted from. After some additional testing, I've found that both WIRED and AWOL content fails to send to draft, while about ten other feeds I tried all sent to draft without difficulty. That said, not all AWOL content failed to send to draft, and none of those that failed to send to draft produced an error message in the console.
